ftw/participation/browser/participants_form.pt

```diff
@@ -7,6 +7,16 @@
     <form action="" method="post"
           tal:attributes="action form_action|string:${here/absolute_url}/@@participants"
           name="manage-participants">
+        <p>
+            <a tal:attributes="href string:${here/absolute_url}/@@invite_participants"
+               i18n:attributes="title link_invite_participants"
+               i18n:translate="link_invite_participants"
+               title="Invite participants"
+               tal:condition="view/can_manage"
+               class="button">
+                Invite participants
```

Comment:
What about `Resend invites`?

Reply:
This is not the Resend Invitations ;-) The right button is at the bottom (Line 84)

Reply:
Ah, sorry I missed that.